### PR TITLE
Tweak banner images and image credits

### PR DESCRIPTION
**Pull request by hatton**
This change is [Reviewable](https://reviewable.io/reviews/bloombooks/bloomlibrary2/256)

### CODE REVIEW

File: src/components/banners/ImageOnRightBannerLayout.tsx [hunk 1]
```diff
@@ -60,36 +55,32 @@ export const ImageOnRightBannerLayout: React.FunctionComponent<{
...
                         // Suzanne & I decided to dial this back to "contain",
                         // and then let css defined in contentful switch back to
                         // "cover" for those images where it works well.
                         background-size: contain;
                         background-repeat: no-repeat;
-                        margin-left: auto; // if contentful then sets the image to have a fixed width, we get right-alignment
+                        // this causes the image to be right aligned.
+                        // margin-left:auto does not work
+                        background-position-x: 100%;
+                        //margin-left: auto; // if contentful then sets the image to have a fixed width, we get right-alignment
```
Comment (gmartin7) [inline src/components/banners/ImageOnRightBannerLayout.tsx:72]: Don't you want to remove the commented `margin-left` rule?

**Review by gmartin7 — COMMENTED** [review]
> Reviewed 4 of 4 files at r1.
> *[Reviewable](https://reviewable.io/reviews/bloombooks/bloomlibrary2/256)* status: all files reviewed, 1 unresolved discussion (waiting on @hatton)

**Review by gmartin7 — APPROVED** [review]
:lgtm: after small comment tweak

> *[Reviewable](https://reviewable.io/reviews/bloombooks/bloomlibrary2/256#-:-MQZ2DSk0tKnDO8r9Dbi:b-sbbyo1)* status: all files reviewed, 1 unresolved discussion (waiting on @hatton)

**Review by andrew-polk — COMMENTED** [review]
> Reviewed 4 of 4 files at r1.
> *[Reviewable](https://reviewable.io/reviews/bloombooks/bloomlibrary2/256)* status: all files reviewed, 2 unresolved discussions (waiting on @hatton)

___
*[src/components/banners/ImageOnRightBannerLayout.tsx, line 59 at r1](https://reviewable.io/reviews/bloombooks/bloomlibrary2/256#-MQZ2Su766iVgAVVYOdU:-MQZ2Su766iVgAVVYOdV:b-h3d06y) ([raw file](https://github.com/bloombooks/bloomlibrary2/blob/78572451748574c7e1ab30ae28e473e9b40f3878/src/components/banners/ImageOnRightBannerLayout.tsx#L59)):*
> ```
> minium
> ```
minimum (if you're making a change anyway)

**Review by hatton — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/bloombooks/bloomlibrary2/256)* status: all files reviewed, 2 unresolved discussions (waiting on @gmartin7 and @hatton)

___
*[src/components/banners/ImageOnRightBannerLayout.tsx, line 72 at r1](https://reviewable.io/reviews/bloombooks/bloomlibrary2/256#-MQZ11OsQYlSd9cPfdox-r1-72:-MQZ7W1e95nUIx64gdH2:bi08a1j) ([raw file](https://github.com/bloombooks/bloomlibrary2/blob/78572451748574c7e1ab30ae28e473e9b40f3878/src/components/banners/ImageOnRightBannerLayout.tsx#L72)):*
<details><summary><i>Previously, gmartin7 (Gordon Martin) wrote…</i></summary><blockquote>

Don't you want to remove the commented `margin-left` rule?
</blockquote></details>

are you just asking to delete something commented? That doesn't seem worth the cycles of a round of prs and commits. But maybe you mean something different?

**Review by gmartin7 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/bloombooks/bloomlibrary2/256)* status: all files reviewed, 2 unresolved discussions (waiting on @hatton)

___
*[src/components/banners/ImageOnRightBannerLayout.tsx, line 72 at r1](https://reviewable.io/reviews/bloombooks/bloomlibrary2/256#-MQZ11OsQYlSd9cPfdox-r1-72:-MQZCC94BJx5zaNN0Jjv:b-24iln4) ([raw file](https://github.com/bloombooks/bloomlibrary2/blob/78572451748574c7e1ab30ae28e473e9b40f3878/src/components/banners/ImageOnRightBannerLayout.tsx#L72)):*
<details><summary><i>Previously, hatton (John Hatton) wrote…</i></summary><blockquote>

are you just asking to delete something commented? That doesn't seem worth the cycles of a round of prs and commits. But maybe you mean something different?
</blockquote></details>

I'll merge it. I just saw that there are 2 lines of margin-left rules commented out.

**Pull request merged**